index.html

```diff
@@ -484,7 +484,7 @@ <h3 class="title">Download NewPipe</h3>
                             <div class="col-xs-12 col-sm-6 col-lg-4" id="download-stable">
                                 <h4>F-Droid</h4>
                                 <div class="download-info">
-                                    <p class="download-version api-f-droid-stable-version">0.11.6</p>
+                                    <p class="download-version api-f-droid-stable-version">0.18.0</p>
```

Comment:
Those data are fetched dynamically via AJAX requests to our API. Hardcoding values in there doesn't make *any* sense really. We should put something in there like "You need to enable JavaScript in order to have up to date information". Otherwise, we need to fetch that information on build and re-render that page occassionally IMO.

CC @TobiGr, perhaps we need to open a new issue about this.

Reply:
a new issue sounds good. I didnt spend time to hunt down that script, but it didnt work on my local website, showed this hardcoded values. Thats why I changed it.

Reply:
> it didnt work on my local website, showed this hardcoded values

That's because of CORS

> CC @TobiGr, perhaps we need to open a new issue about this.

Yes, there are some more things to consider before displaying a message like "You need to enable JavaScript in order to have up to date information". For example this long text cannot be displayed on the button because it would destroy the layout.

Reply:
One might just show nothing or a `-` or `?` or something.

Reply:
What about a question mark with the long JS required message as hover attribute?

Reply:
I'd like to keep default values when possible. This works e.g. when displaying the GitHub stars (7100+) or supported languages (60+), but is completely non-sense when displaying  the current version. In this cases, we should try to link to sites where the information is also displayed like the GitHub releases page.

Please keep also in mind that it might be our fault that the api information is not available. In this case all users would see the "please enable javascript" message.

Hover titles are bad for users on touch devices. In this case we would need to display an icons to tell them that there is more info to be displayed when tapping on the text.

Reply:
Can I just put an a tag inside the p, without breaking the replacement from the javascript? Then I'd have no problem linking that.

Reply:
It is okay for now because we replace the `innerHTML` as you can see [here](https://github.com/TeamNewPipe/website/blob/fc735a839feef1527d746f9b801074dc33554a3f/js/web-api.js#L34). I have aslo implemented the option to add api-data as attribute, but this is not used atm.

Reply:
okay, I will do this then